reflex/state.py

```diff
@@ -1532,14 +1539,22 @@ async def _process_event(
                 yield state._as_state_update(handler, events, final=True)
 
         # If an error occurs, throw a window alert.
-        except Exception:
-            error = traceback.format_exc()
-            print(error)
-            yield state._as_state_update(
-                handler,
-                window_alert("An error occurred. See logs for details."),
-                final=True,
-            )
+        except Exception as e:
+            stack_trace = traceback.format_exc()
+
+            config = get_config()
+
+            # Call the custom backend exception handler if specified. If not, fallback to the default handler.
```

Comment:
How about just passing the whole Exception (variable `e`) to the exception handlers instead of only passing 2 strings?